omnibus/files/private-chef-cookbooks/private-chef/resources/chef_run.rb

```diff
@@ -0,0 +1,58 @@
+provides :chef_run
```

Comment:
We might want to explicitly test if this thing still works manually.  It is pretty tightly coupled to chef-client internals....

Reply:
Testing a plugin install should be sufficient, right?